tests/foreman/api/test_docker.py

```diff
@@ -453,9 +452,31 @@ def test_add_multiple_docker_repos_to_content_view(self):
 
         @Feature: Docker
 
-        @Status: Manual
-
         """
+        upstream_name = u'busybox'
+        product = entities.Product(organization=self.org).create()
+        repos_amount = randint(1, 5)
+        repos = [
+            _create_repository(product, gen_string('alpha'), upstream_name)
+            for _
+            in range(repos_amount)
+        ]
+        self.assertEqual(len(repos), repos_amount)
+
+        content_view = entities.ContentView(
+            composite=False,
+            organization=self.org,
+        ).create()
+        content_view.repository = repos
+        content_view = content_view.update(['repository'])
+
+        self.assertEqual(len(content_view.repository), repos_amount)
+        initial_repos_ids = [repo.id for repo in repos]
+        for repo_ in content_view.repository:
+            self.assertIn(repo_.id, initial_repos_ids)
+            repo = entities.Repository(id=repo_.id).read()
```

Comment:
`content_view.repository` is a list of `entities.Repository` objects, and each one has a `content_type` and `docker_upstream_name` attribute. As a result, you can operate with those fields from `content_view.repository` point of view.